### ISSUE TITLE
If you have crumbs of native token on your balance, the alt fee asset will not be used 

### ISSUE THREAD

**Issue by Valentine1898** [penumbra-zone/web#1595]
**Describe the bug**
If you have a small native token balance that is not enough to pay the fee, you will not be able to use alt fee assets

**Expected behavior**
The user should be able to manually select the fee asset in this case

**Screenshots**
<img width="754" alt="image" src="https://github.com/user-attachments/assets/7d678104-d4a9-4bf7-baab-1e0c8c572964">
<img width="360" alt="image" src="https://github.com/user-attachments/assets/71d2cdee-50fa-43fb-a4c3-14e22cea5f26">

**Comment by TalDerei** [comment]
this is something we should definitely follow-up with on.

**Comment by hdevalence** [comment]
Yes, but lower priority than making sure that fees work when there are no native tokens.

**Comment by TalDerei** [comment]
From a ux perspective, one alternative could be to define a custom rust error type, detect the error, propagate it up the call stack to the caller, and handle it in typescipt by automatically retrying the transaction with another fee asset. But replanning a transaction and retriggering the popup doesn't seem like a good idea to me. Just a thought

**Comment by TalDerei** [comment]
do we want to implement manual fee selection as a feature?

**Comment by hdevalence** [comment]
Don't think so, we can revisit this as soon as a single user complains about it

**Comment by TalDerei** [comment]
moving back to backlog for now
